src/main/java/com/minecolonies/coremod/items/ItemScepterPermission.java

```diff
@@ -122,7 +122,7 @@ public EnumActionResult onItemUse(
     {
         if (worldIn.isRemote)
         {
-            return new ActionResult(EnumActionResult.SUCCESS, scepter);
+            return new ActionResult<ItemStack>(EnumActionResult.SUCCESS, scepter);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2293)